storage/entities/transaction.js

```diff
@@ -684,6 +492,39 @@ class Transaction extends BaseEntity {
 
 		return filters;
 	}
+
+	static _sanitizeCreateData(data) {
```

Comment:
I would've preferred normal function instead of `static` but I see another `static` function above ='(